web/cobrands/sass/_layout.scss

```diff
@@ -889,12 +889,19 @@ textarea.form-error {
 }
 
 .alerts__columns {
-    @include clearfix;
-    @include flex-container();
-    > div {
-        width: 50%;
-        margin: 0 2em;
-    }
+  display: grid;
+  // This tells the column to have a minimum size(23.75rem, around 380px), if it can't be then allows it to take the whole space available.
+  grid-template-columns: repeat(auto-fit, minmax(23.75rem, 1fr));
+  column-gap: 5rem;
+
+  &::before, &::after {
+    position: absolute;
```

Comment:
I'm not sure what this is for?

Reply:
We don't need it anymore, before those two elements were being treated as Childs for the grid parent, and using space accordingly. But now we are not using the `clearfix` mixin we were using before, so those pseudo elements are not being created.